src/_P004_Dallas.ino

```diff
@@ -154,7 +154,8 @@ boolean Plugin_004(byte function, struct EventStruct * event, String& string)
 
                 if (Plugin_004_DS_readTemp(addr, &value))
                 {
-                    UserVar[event->BaseVarIndex] = value;
+                    if (value != 85.0)    // ignore power on reset value
```

Comment:
Do not compare a float like that. You cannot compare floats using `==` or `!=`.
Better to check for errors in the read function `Plugin_004_DS_readTemp` itself and return false when an error occurred.